src/types/ProfileForm/type.ts

```
@@ -0,0 +1,23 @@
+  employmentStatus: string;
+  homePhoneNumber: string;
+  maritalStatus: string;
+  mobilePhoneNumber: string;
+  name: string;
+  natalSex: string;
+  naturalLanguageCode: string;
+  password: string;
+  postalCode: string;
+  state: string;
+  workPhoneNumber: string;
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)